src/components/core/DpTreeList/DpTreeList.vue

```diff
@@ -81,6 +81,7 @@
 </template>
 
 <script>
+import bus from './utils/bus'
```

Comment:
Nitpick: alphabetic order.

Reply:
true, done in [7d116bb](https://github.com/demos-europe/demosplan-ui/pull/73/commits/7d116bb84e96528540a82036a9e9c258a2869af1)